docs/metrics/README.md

```diff
@@ -403,13 +403,9 @@ SDK to reclaim unused metric points.
 
 > [!NOTE]
 > Reclaim unused metric points feature was introduced in OpenTelemetry .NET
-  [1.7.0-alpha.1](../../src/OpenTelemetry/CHANGELOG.md#170-alpha1). It is
-  currently an experimental feature which can be turned on by setting the
-  environment variable
-  `OTEL_DOTNET_EXPERIMENTAL_METRICS_RECLAIM_UNUSED_METRIC_POINTS=true`. Once the
-  [OpenTelemetry
-  Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute)
-  become stable, this feature will be turned on by default.
+  [1.7.0-alpha.1](../../src/OpenTelemetry/CHANGELOG.md#170-alpha1) as an
+  experimental feature, and become stable and the default behavior starting with
+  version 1.10.0.
```

Comment:
Not sure everyone will agree, but I think we need to keep the instructions for enabling it on older versions. Reason being, some users might not be able or willing to upgrade to latest right away?

Suggestion (starting a bit higher up than the diff)...

When [Delta Aggregation Temporality](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#temporality) is used, it is possible to choose a smaller cardinality limit because the SDK will automatically reclaim unused metric points.

> [!NOTE]
> Metric point reclaim is enabled automatically starting with the `1.10.0` SDK. It can be enabled in `1.7.0` - `1.9.0` SDKs by setting the environment variable `OTEL_DOTNET_EXPERIMENTAL_METRICS_RECLAIM_UNUSED_METRIC_POINTS=true`.

Reply:
yes this is reasonable.

Reply:
I don't have strong opinion on this. I put it this way because for the overflow attribute promotion a few lines above, we only mentioned that there were changes in README and kept details of previous/new behavior in CHANGELOG.

Reply:
Updated.

Reply:
I tried to make the two sections a bit more consistent on #5957